Tester/Runner/CliTester.php

```diff
@@ -222,7 +229,7 @@ private function prepareCodeCoverage()
 	/** @return void */
 	private function finishCodeCoverage($file)
 	{
-		if ($this->options['-o'] !== 'none' && $this->options['-o'] !== 'tap') {
+		if ($this->options['-o'] !== 'none' && $this->options['-o'] !== 'tap' && $this->options['-o'] !== 'junit') {
```

Comment:
~~~php
if (!in_array($this->options['-o'], array('none', 'tap', 'junit'), TRUE) {
~~~